### DISCUSSION TITLE
Flatpak version for the unexperienced user?

### DISCUSSION THREAD

**Discussion by fccfc5d245c** [tsujan/Kvantum#1020] [Ideas]
Now that the easy way of installing Kvantum doesn't work (it was the PPA method, if I remember correctly), lots of unexperienced users will loose access to this software. Maybe packaging it on Flatpak would solve the issue. 
I could post a petition on Flathub.

PS: I intend to figure out how to compile and install it myself, but I suspect many users won't try.

**Comment by tsujan** [comment]
> Now that the easy way of installing Kvantum doesn't work...

That may be true only for distros like Ubuntu.

> Maybe packaging it on Flatpak would solve the issue.

Flatpak usually creates hassles for its users; often this or that thing doesn't work as it should.

> I intend to figure out how to compile and install it myself

Unless you're familiar with cmake and have a good knowledge of your distro, direct installation from the source is **not** recommended.

The best way is always *making an installable package from the source* and then installing it. I do that too because only in this way the package manager is aware of the installation.

In Arch and its derivatives it's a piece of cake (although Arch has the latest version of Kvantum); Debian has its own way of deb making, which is relatively easy and may be inherited by Ubuntu; in Fedora you could make an rpm out of a source; etc. Consult you distro's doc.

**Reply by fccfc5d245c** [reply]
>  That may be true only for distros like Ubuntu.

Exactly, there's where I guess most newbies are. I wrote this idea because I think the installation process, being a bit more complex than usual in some distros, will deter that kind of user, which i's a pity, because they would love Kvantum.

> Flatpak usually creates hassles for its users; often this or that thing doesn't work as it should.

As a newbie myself, Flatpak has worked fine for me and I thought it would be the easier option, but I'm sure you are right.

Thanks for the info! I wanted to throw myself into the deep end so I'm forced to learn more, but I'll start by following your advice.
